labyrwa/wedding
# Homepage

Let me know what you think of the whole map thing, if you think it's cool I can make the font more readable, but before I do that I want to make sure that it's worth it. Thanks.

Testing instructions:
- pull code
- run `gulp develop` in terminal
- check homepage in browser
- observe the map/event detail area at the bottom of the homepage
